tasks/tests/test_view_all_tasks.py

```diff
@@ -40,7 +40,7 @@ def test_filter_task_by_priority(self, client, test_db):
         response = client.post('/tasks/', data={'priority': 'Low'})
         assert all(task.priority == Priority.LOW for task in response.context['tasks'])
 
-    @pytest.mark.parametrize('flag, priority_text',
+    @pytest.mark.parametrize('EmpOrMan, priority_text',
```

Comment:
The pythonic way is with an underline:
```suggestion
    @pytest.mark.parametrize('employee_or_manager, priority_text',
```
"emp_or_man" IMHO is not readable 
so I suggested all words like "employee_or_manager" 
but you can do it as you want maybe "user_type"